mysql-test/t/order_by_sortkey.test

```diff
@@ -78,3 +78,16 @@ SELECT * FROM t2 where f1 =
 SHOW SESSION STATUS LIKE 'Sort%';
 
 DROP TABLE t1, t2, tmp;
+
+--echo #
+--echo # BUG: PS-7221 - crash on select Sys_var_tz::session_value_ptr
+--echo #
+
+CREATE TABLE t1 (
+  id INT UNSIGNED NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  dt DATETIME NOT NULL
+);
+INSERT INTO t1 SET dt=NOW();
+INSERT INTO t1 (dt) SELECT NOW() FROM t1;
```

Comment:
Do we need second record to be inserted to reproduce this bug?

Reply:
@percona-ysorokin yes, without 2 rows to sort, we will not hit the path that leads to the assertion.